meetings/2023/2023-04-25.md

```diff
@@ -0,0 +1,88 @@
+# OpenJS Foundation Cross Project Council Meeting 2023-04-25
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=qQFW9k6MfNc
+* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1058
+
+## Present
+
+* Joe Sepi (@joesepi)
+* Abby Cabunoc Mayes (@abbycabs)
+* Ben Sternthal (@bensternthal)
+* Ruy Adorno (@ruyadorno)
+* Yagiz Nizipli (@anonrig)
+* Tobie Langel (@tobie)
+* Christian Bromann <@christian-bromann)
+* Michael Dawson (@mhdawson)
+* Tierney Cyren (@bnb)
+
```

Comment:
I was very, very late, but I was in attendance. It's OK if it doesn't count--I don't know what the rules and norms are around attending only part of the meeting and being marked as present in the minutes--but if it was an oversight....

```suggestion
* Rich Trott (@Trott)

```

Reply:
> I don't know what the rules and norms are around attending only part of the meeting and being marked as present in the minutes

I'd argue that if you make it on time to put your name in the doc, you should be good, so essentially showing up is fine. Also we don't really have or enforce a policy around good standing.